packages/resources/carbon.yml

```diff
@@ -283,3 +290,59 @@ designKits:
     url: https://www.figma.com/file/upZIO5LXG9OnHCKwCawlur/Light---Cloud-PAL
     action: link
     noIndex: true
+  figma-v11-white-theme:
+    name: Figma v11 white theme
+    tool: figma
+    type: ui
+    status: stable
+    url: https://www.figma.com/file/YAnB1jKx0yCUL29j6uSLpg/(v11-BETA)-White-Theme---Carbon-Design-System
+    action: link
+  figma-v11-gray-10-theme:
+    name: Figma v11 gray 10 theme
+    tool: figma
+    type: ui
+    status: stable
+    url: https://www.figma.com/file/fH8rkv1VUTw5pvFDvO1kPO/(v11-BETA)-Gray-10-Theme---Carbon-Design-System
```

Comment:
For some reason this link doesn't work for me-- it switches to the gray 90 theme upon loading the link. Try this link instead:

https://www.figma.com/file/0sd447WTcazDI6Dq9tjXOe/(v11-BETA)-Gray-10-Theme---Carbon-Design-System?node-id=58%3A2763

Reply:
we had the wrong link - fixed it!